tests/integration/security/filebased_tls_origination/egress_gateway_origination_test.go

```diff
@@ -370,15 +327,25 @@ spec:
         name: http-port-for-tls-origination
         protocol: HTTP
       hosts:
-        - some-external-site.com
+        - server.{{.ServerNamespace}}.svc.cluster.local
 ---
 apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: egressgateway-for-server
+spec:
+  host: istio-egressgateway.istio-system.svc.cluster.local
+  subsets:
```

Comment:
why do we need this subset?

Reply:
just being consistent with istio.io docs which uses subsets

Reply:
Also I would say having a subset is strictly more rigorous test for CDS generation